default/scripting/specials/planet/monster_nest/monster_nest.macros

```diff
@@ -11,7 +11,8 @@ MONSTER_NEST
                 Planet
                 OwnedBy affiliation = AnyEmpire
                 OwnerHasTech name = "SHP_DOMESTIC_MONSTER"
-                Random probability = @3@
+                Random probability = @3@ * (GameRule name = "RULE_DOMESTIC_NEST_MONSTER_SPAWN_FACTOR")
+                ((GameRule name = "RULE_NESTS_ALWAYS_SPAWN_WILD") = 0)
```

Comment:
switch the positions of the lines - it will process faster in a non-meaningful way

Reply:
For an activation condition, putting the `GameRule` condition might be slightly faster, but probably not noticably so.

https://github.com/freeorion/freeorion/blob/master/universe/Universe.cpp#L1456

It assembles a container of all objects with a special, then tests the activation condition on that container of objects. There's probably not many such objects, so it won't make much difference how long it takes to evaluate that condition, but it does early abort if nothing remains in the candidates after evaluating some of the subconditions of `And`.

Reply:
Switched the order. Sounds like its not a big deal either way, but its easy enough to move